hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -212,5 +212,15 @@
             timeout: 300
             emails: '$DEFAULT_RECIPIENTS, ihmccreery@google.com'
             test-owner: 'ihmccreery'
+        - 'gke-ingress':
```

Comment:
`e2e.sh` declares Bash variables & configuration; this file tells Jenkins to actually run a job with this name.